install_config/configuring_local.adoc

```diff
@@ -295,18 +311,18 @@ data:
             hostDir:  /mnt/local-storage/block-devices <2>
             mountDir: /mnt/local-storage/block-devices <3>
 ----
-<1> Name of the StorageClass.
+<1> Name of the storage class.
 <2> Path to the directory on the host. It must be a subdirectory of *_/mnt/local-storage_*.
-<3> Path to the directory in the provisioner pod. We recommend using the same directory structure as used on the host and `mountDir` can be omitted in this case.
-. Change the SELinux label of the device and the *_/mnt/local-storage/_*:
+<3> Path to the directory in the provisioner pod. It is recommended to use the same directory structure as used on the host; `mountDir` can be omitted in this case.
```

Comment:
s/It is recommended to use the same directory structure as used on the host; `mountDir` can be omitted in this case./If you use the directory structure that the host uses, which is recommended, omit the `mountDir` parameter.